packages/ERTP/src/issuer.js

```diff
@@ -297,3 +295,17 @@ function makeIssuerKit(allegedName, mathHelpersName = 'nat') {
 harden(makeIssuerKit);
 
 export default makeIssuerKit;
```

Comment:
let's not `export default` and `export`. I can see why we might want to for backwards compatibility, but here it looks like we are doing so on purpose. Can we move `makeLocalAmountMath` out of this file and create an `index.js` instead that imports and exports both?

Reply:
done